components/atoms/Heading/HeadingH4.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const HeadingH4 = (props) => {
```

Comment:
Can we create one common component for heading and pass h1 or h2 as a component in it. styling can be appended based on the passed heading type.